example/src/App.tsx

```diff
@@ -1,12 +1,68 @@
 import { useState, useEffect } from 'react';
-import { StyleSheet, View, Text } from 'react-native';
-import { multiply } from 'posthog-react-native-session-replay';
+import { StyleSheet, View, Text, Platform } from 'react-native';
+import type PostHogReactNativeSessionReplay from 'posthog-react-native-session-replay';
+
+export let OptionalReactNativeSessionReplay:
+  | typeof PostHogReactNativeSessionReplay
+  | undefined;
+
+try {
+  OptionalReactNativeSessionReplay = Platform.select({
+    macos: undefined,
+    web: undefined,
+    default: require('posthog-react-native-session-replay'), // Only Android and iOS
+  });
+} catch (e) {
+  // do nothing
+  console.warn(
+    'PostHog Debug',
+    `Error loading posthog-react-native-session-replay: ${e}`
+  );
+}
 
 export default function App() {
-  const [result, setResult] = useState<number | undefined>();
+  const [result, setResult] = useState<string | undefined>();
 
   useEffect(() => {
-    multiply(3, 7).then(setResult);
+    if (OptionalReactNativeSessionReplay) {
+      setResult('ok');
+      // OptionalReactNativeSessionReplay.isEnabled().then((isEnabled) => {
+      //   console.warn('PostHog Debug', `isEnabled: ${isEnabled}`);
+      //   setResult(isEnabled.valueOf().toString());
+      // });
+      // OptionalReactNativeSessionReplay.startSession(
+      //   'e58ed763-928c-4155-bee9-fdbaaadc15f3'
+      // )
+      //   .then(() => {
+      //     setResult('ok');
+      //   })
+      //   .catch(() => {
+      //     setResult('failed');
+      //   });
+      // OptionalReactNativeSessionReplay.start(
+      //   'e58ed763-928c-4155-bee9-fdbaaadc15f3',
+      //   {
+      //     apiKey: 'phc_QFbR1y41s5sxnNTZoyKG2NJo2RlsCIWkUfdpawgb40D',
+      //     host: 'https://us.i.posthog.com',
+      //   },
+      //   {},
+      //   {}
+      // )
+      //   .then(() => {
+      //     OptionalReactNativeSessionReplay?.isEnabled().then((isEnabled) => {
+      //       console.warn('PostHog Debug', `isEnabled: ${isEnabled}`);
+      //       setResult(`isEnabled: ${isEnabled}`);
+      //     });
+      //   })
+      //   .then(() => {
+      //     setResult('ok');
+      //   })
+      //   .catch(() => {
+      //     setResult('failed');
+      //   });
```

Comment:
```suggestion
```

Reply:
I'd rather keep it since it's for testing the native calls (during development - its just a sample), ideally I'd have a button for each of them, will do it in the next PR with iOS if that's ok